.github/CONTRIBUTING.md

```diff
@@ -12,21 +12,30 @@ Before you begin, check out our [Developer Guidelines](https://www.jenkins.io/do
 
 ### Issues
 
-#### Create a new Issue
+#### Create a New Issue
 
 If you spot an issue with this project, [search if an issue already exists](https://github.com/jenkinsci/conventional-commits-plugin/issues). If a related issue doesn't exist, you can open new issue using a relevant [issue form](https://github.com/jenkinsci/conventional-commits-plugin/issues/new/choose).
 
 #### Solve an Issue
 
 Go through our existing issues to find one that interests you. You could also use `labels` as filters, to find the best issue for you.
 
+### Run the plugin
+
+To run the plugin, use the following command, as described in the [official documentation](https://www.jenkins.io/doc/developer/tutorial/run/) on running a plugin:
+```shell
+mvn hpi:run
+```
+
+For this to be successful, please make sure Jenkins is already installed but not already running. The default URL where Jenkins is set up in the browser is [http://localhost:8080/jenkins/](http://localhost:8080/jenkins/), where Jenkins can be accessed once up and running. 

```

Comment:
for me it's not mandatory to have a version of Jenkins installed to work.
I do not have a Jenkins installed (I use dicker image to launch a Jenkins instance) and the `mvn hpi:run`work.

Reply:
How about if I change it up a bit and add something to the effect of "... please make sure to either have Jenkins installed, or a docker image of Jenkins is available for launch"?

Reply:
yes perfect !
I agree with you that it's better to have a real instance to do some real tests !

Reply:
Sure, let me make the change now